src/Shared/AzureManaged/GrpcRetryPolicyDefaults.cs

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Grpc.Core;
+using Grpc.Net.Client.Configuration;
+
+namespace Microsoft.DurableTask;
+
+/// <summary>
+/// Provides default retry policy configurations for gRPC client connections.
+/// </summary>
+sealed class GrpcRetryPolicyDefaults
```

Comment:
I am wondering is this class intentionally being non-public for customer configuration but just for internal use ? I asked this mainly because the class is sealed, while the properties are public.

Reply:
for internal use only, we dont want customer to override retry configs